pkg/helm/values/eks.go

```diff
@@ -59,18 +57,10 @@ func getDefaultEKSReleaseValues(chartOptions *helm.ChartOptions, log log.Logger)
 	}
 
 	// build values
-	values := `api:
-  image: ##API_IMAGE##
-controller:
-  image: ##CONTROLLER_IMAGE##
-etcd:
-  image: ##ETCD_IMAGE##
-coredns:
-  image: ##COREDNS_IMAGE##
-`
-	values = strings.ReplaceAll(values, "##API_IMAGE##", apiImage)
-	values = strings.ReplaceAll(values, "##CONTROLLER_IMAGE##", controllerImage)
-	values = strings.ReplaceAll(values, "##ETCD_IMAGE##", etcdImage)
-	values = strings.ReplaceAll(values, "##COREDNS_IMAGE##", corednsImage)
+	values := ""
+	values += "api.image=" + apiImage
+	values += ",coredns.image=" + corednsImage
+	values += ",controller.image=" + controllerImage
+	values += ",etcd.image=" + etcdImage
```

Comment:
should we use a `Join` here instead?

Reply:
Yea what I meant was to probably have slice of strings instead and finally join it just before using. Something like
```
values := []string{}

apiImage := "image"
corednsImage := "corednsImage"
controllerImage := "controllerImage"
etcdImage := "etcdImage"

values = append(values,
	"apiImage="+apiImage,
	"coredns.image="+corednsImage,
	"controller.image="+controllerImage,
	"etcd.image="+etcdImage,
)
...
...
...


// finally right before using values
strings.Join(values, ",")
```

that way we can avoid the complications of where a leading `,` separator is required or not